src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component,OnInit,AfterViewInit, } from "@angular/core";
+import { EntityMapperService } from "../../../../core/entity/entity-mapper.service";
+import { Child } from "../../model/child";
+import { DynamicComponent } from "../../../../core/view/dynamic-components/dynamic-component.decorator";
+import { OnInitDynamicComponent } from "../../../../core/view/dynamic-components/on-init-dynamic-component.interface";
+import { MatTableDataSource } from "@angular/material/table";
+import { MatSort } from "@angular/material/sort";
+import { MatPaginator } from "@angular/material/paginator";
+import { ViewChild } from "@angular/core";
+import { ChildrenBmiDashboardComponent } from "../children-bmi-dashboard/children-bmi-dashboard.component";
+import childBlockStories from "../../child-block/child-block.stories";
+
+@DynamicComponent("BirthdayDashboard")
+@Component({
+  selector: "app-birthday-dashboard",
+  templateUrl: "./birthday-dashboard.component.html",
+  styleUrls: ["./birthday-dashboard.component.scss"],
+})
+export class BirthdayDashboardComponent implements OnInitDynamicComponent,AfterViewInit {
+
+  children: Child[] = [];
+  displayedColumns: string[] = ['entityid', 'dateOfBirth', 'age'];
```

Comment:
```suggestion
  displayedColumns: string[] = ['entityId', 'dateOfBirth', 'age'];
```
There is also a error message in the console in the browser which says it can't find the column named `entityid` which indicates this typo.